starboard/shared/starboard/player/player_worker.h

```diff
@@ -111,6 +111,8 @@ class PlayerWorker {
 
     virtual SbDecodeTarget GetCurrentDecodeTarget() = 0;
 
+    virtual void SetMaxVideoInputSize(int max_video_input_size) = 0;
```

Comment:
It's not used.

Reply:
It is implemented in `starboard/shared/starboard/player/filter/filter_based_player_worker_handler.h`.